### PR TITLE
[313] Careers page feedback

### PR DESCRIPTION
**Pull request by chris-lawton**
https://projects.torchbox.com/projects/tbxcom/tickets/313

Incudes:
- Adding gradient to hero image
- Include the card links at the top and bottom of the page
- Swap the order of the title and subtitle of standout items
- Remove body streamfield, contact block and strapline visible fields

### CODE REVIEW

**Comment by tbrlpld** [comment]
**Suggestion** I can mention lines that where not changes in the PR, but Flake8 is complaining about unused imports:

```
tbx/people/models.py:23:1: F401 'wagtail.snippets.edit_handlers.SnippetChooserPanel' imported but unused
tbx/people/models.py:27:1: F401 'tbx.core.blocks.StoryBlock' imported but unused
```

These imports can be remove.

File: tbx/static_src/sass/components/_hero.scss [hunk 1]
```diff
@@ -112,5 +112,18 @@
                 min-height: 700px;
             }
         }
+
+        &--image-overlay {
```
Comment (tbrlpld) [inline tbx/static_src/sass/components/_hero.scss:116]: **Question** I am totally new to BEM. I though `--` is for varying status like "hover" etc? Do we also use it as a component separator?

**Review by tbrlpld — COMMENTED** [review]
Only change necessary is removing the unused import to fix the CI. Apart from that looks good to me 👍

File: tbx/static_src/sass/components/_hero.scss (hunk 1 above)
Comment (chris-lawton) [reply tbx/static_src/sass/components/_hero.scss:116]: At Torchbox we tend to just stick with using `:hover` and `:focus` for styles relating to those states.

The `--` in BEM represents a modifier, which signifies an adjustment in styles on a Block or Element. In this instance i'm adding a modifier on the `hero__full-screen` element, so the final class is `hero__full-screen--image-overlay`.
https://css-tricks.com/bem-101/ is a good intro : )

**Comment by chris-lawton** [comment]
Thanks @tbrlpld!

File: tbx/static_src/sass/components/_hero.scss (hunk 1 above)
Comment (tbrlpld) [reply tbx/static_src/sass/components/_hero.scss:116]: Thanks @chris-lawton. That makes sense.

**Pull request merged**